### DISCUSSION TITLE
Not able to import CISA Alert RSS Feed.

### DISCUSSION THREAD

**Discussion by pmisner** [FreshRSS/FreshRSS#5961] [Q&A] [answered]
I have a website with FreshRSS installed at https://paulmisner.com/rss It is for infosec research, One of the essential sites I need is CISA's feed, https://www.cisa.gov/cybersecurity-advisories/all.xml When I try to ingest the feed, I get the following error message. "The newsfeed could not be added. Check FreshRSS logs for details. You can try force adding by appending #force_feed to the URL." I tried the "force_feed" append, and get the same error. 
The logs have two entries related to this:
cURL error 22: The requested URL returned error: 403 [https://www.cisa.gov/cybersecurity-advisories/all.xml]
 XML+XPath parsing failed for [https://www.cisa.gov/cybersecurity-advisories/all.xml]

I am able to ingest the feed at FeedReader.com/online. I've invested a bit of time in FreshRSS and don't want to leave it, but it is essential that we are able to get THIS feed installed. I am using version 1.22.1

Thank you in advance. I welcome any suggestions.

**Comment by Alkarex** [comment, marked as answer]
https://www.cisa.gov/cybersecurity-advisories/all.xml works out of the box in my FreshRSS instance.
You should not use `XML+XPath` but just default RSS mode.
If you can, try from your server:

```sh
curl -v 'https://www.cisa.gov/cybersecurity-advisories/all.xml'
```

A 403 can be a sign that your IP (or IP range) has been banned.

If that is the case, you have two main options:
* Move your FreshRSS instance to another IP, for instance at home;
* Use a proxy for your (problematic) feeds, the proxy being hosted elsewhere.

**Reply by pmisner** [reply]
Thank you for the tip. I'm hosted in Germany and I'm wondering if they geoblock there. It's a shared server I don't have command prompt, but I requested that support run the command.

**Reply by pmisner** [reply]
I decided to take your advice and moved it to a small Linode.  It's working well, but I am having issues with firewalling. I'll post my question in another thread.
